modules/installation-osp-describing-cloud-parameters.adoc

```diff
@@ -41,7 +42,34 @@ clouds:
       auth_url: 'https://10.10.14.22:5001/v2.0'
 ----
 
-. Place the file that you generate in one of the following locations:
+. _Optional_: If your {rh-openstack} installation uses self-signed certificate authority (CA) certificates for endpoint authentication:
```

Comment:
We don't add styling to "Optional."

If you have to do this process if you meet these criteria, it's not optional.